.travis.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     - stage: test
       script: tools/run_in_docker "bazel --bazelrc=tools/travis.bazelrc build //particles/... && bazel --bazelrc=tools/travis.bazelrc test //particles/... && tools/sigh webpackStorage && tools/sigh testShells"
       name: "Bazel: //particles/... + testShells"
-    - script: tools/run_in_docker "bazel --bazelrc=tools/travis.bazelrc build //java/... //javatests/... && bazel --bazelrc=tools/travis.bazelrc test //java/... //javatests/..."
+    - script: tools/run_in_docker "bazel --bazelrc=tools/travis.bazelrc build --spawn_strategy=standalone //java/... //javatests/... && bazel --bazelrc=tools/travis.bazelrc test --spawn_strategy=standalone //java/... //javatests/..."
```

Comment:
I'm not sure exactly what this `--spawn_strategy=standalone` does. Is it still needed? Is it useful for the other bazel invocations? If so, it might be better to put it in the travis.bazelrc file.